index.bs

```diff
@@ -282,6 +282,12 @@ type: dfn
           <li>
             <var>request</var>'s <a for="request">destination</a> is "<code>image</code>"
             and <var>request</var>'s <a for="request">initiator</a> is "<code>imageset</code>".
+
+            Note: For historical reasons, "<code>imageset</code>" is not upgraded. Prior to some
+            types of mixed content being upgraded, the spec divided mixed content into blockable
+            and optionally-blockable. "<code>imageset</code>" was part of the former, and when
```

Comment:
suggest adding "blockable and optionally-blockable, with blockable being preferred for any type of mixed content that wasn't prevalent."

Reply:
Done, thanks for reviewing.